packages/zoe/src/zoeService/zoe.js

```diff
@@ -291,7 +291,7 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
         },
         shutdown: () => {
           exitAllSeats();
-          adminNode.terminate();
+          E(adminNode).terminate();
```

Comment:
good catch!

Reply:
Don't give me too much credit. A test failed since adminNode is a presence and doesn't have a terminate() method of its own.